pokete.py

```diff
@@ -496,7 +497,7 @@ def set_vars(self):
         for name in ["atc", "defense", "initiative"]:
             setattr(self, name, self.lvl() + self.inf[name]
                     + (2 if self.shiny else 0))
-        i = [Attack(atc)
+        i = [Attack(atc, p_data)
```

Comment:
Change unnecessary (see comment in attack.py line 4).

Reply:
No, the `p_data` imported is not the same as the injected one. Because mods are able to modify `p_data` it has to be injected so the changes there won't be discarded.

Reply:
Ok, so maybe add a a _docstring_ comment to Attack and Types like the following to elaborate this further:

For types:
```python
class Types:
    """Class to organize PokeTypes"""

    def __init__(self, p_d):
        """Initializes a pokete type for use

        Parameters
        ----------
        p_d (by mods modified) pokete_data
        """
        for i in p_d.types:
            setattr(self, i, PokeType(i, **p_d.types[i]))
        for i in p_d.sub_types:
            setattr(self, i, PokeSubType(i))
```

or for attacks:
```python
    def __init__(self, index, p_d):
        """Initializes an attack by its index for further use cases

        Parameters
        ----------
        index the index of the attacs dictionary
        p_d (by mods modified) pokete_data
        """
        inf = p_d.attacks[index]
        # Attributes
        self.name = inf["name"]
        [...]
```

Reply:
I will go with a comment in `pokete_data` to never ever import it except for in `pokete.py` because `p_data` has to be modified by mods before.

Reply:
Nice, but...

More work for the person(s) who is/are documenting all the functions and methods in the future...